hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -135,6 +136,9 @@ public synchronized void open() throws IOException {
         hConfig.setInt("hbase.hconnection.threads.max",
                        config.get(HbaseOptions.HBASE_THREADS_MAX));
 
+        String hbaseSite = config.get(HbaseOptions.HBASE_HBASE_SITE);
```

Comment:
can it be null? don't addResource if hbaseSite is null

Reply:
This config item's default value is /etc/hbase/conf/hbase-site.xml.If not setting,program will get default value.

Reply:
what happens if there not exist file "/etc/hbase/conf/hbase-site.xml"

Reply:
The program runs normally.If not modified,program read the file from hbase-common-2.0.6.jar, it does not exist actually but prograpm runs normally.

```
  public static Configuration addHbaseResources(Configuration conf) {
    conf.addResource("hbase-default.xml");
    conf.addResource("hbase-site.xml");

    checkDefaultsVersion(conf);
    return conf;
  }
```